drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -209,7 +220,7 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_item.data_source_id = enrollment.data_source_id
       report_item.project_id = enrollment.project.id
       report_item.project_name = enrollment.project.name(report.user)
-      project_type =  enrollment.project.project_type_to_use
+      project_type = enrollment.project.project_type_to_use
```

Comment:
this can just be `enrollment.project.project_type` now.  (With the last iteration of overrides, `project_type_to_use` became a long convoluted way of getting to `project_type`.)

Reply:
will include this in the next commit